qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,108 @@
+Feature: Sending CV
```

Comment:
you need to specify the file number and Feature, in the future it will be convenient and orderly. For example, 1.2.1 Send_CV.feature.